tools/pythonpkg/tests/extensions/substrait/test_duckdb_ibis_tpch.py

```diff
@@ -0,0 +1,123 @@
+import duckdb
+import pytest
+
+try:
+    from ibis_substrait.compiler.decompile import decompile
+    from ibis_substrait.proto.substrait import plan_pb2
+    can_run = True
```

Comment:
If you're rewriting these tests anyway, can you please use the `importorskip` function and remove the try-catch and `can_run` pattern?

Reply:
rewriting is a strong word, but will do, I forgot about that ;)